models/contacts.js

```diff
@@ -1,19 +1,126 @@
-// const fs = require('fs/promises')
+const { readFile, writeFile } = require("fs/promises");
+const { v4: uuidv4 } = require("uuid");
+const path = require("path");
+require("colors");
 
-const listContacts = async () => {}
+const baseDir = __dirname;
+const fileName = "contacts.json";
+const contactsPath = path.join(baseDir, fileName);
```

Comment:
Jest prostsze, mniej kodu :)
```suggestion
const contactsPath = path.join(__dirname, fileName);
```